sematic/db/models/tests/test_resolution.py

```diff
@@ -24,72 +25,104 @@ def test_is_allowed_transition():
     )
 
 
+def make_resolution(root_id, status, kind, docker_image_uri, git_info):
+    resolution = Resolution(
+        root_id=root_id,
+        status=status,
+        kind=kind,
+        docker_image_uri=docker_image_uri,
+    )
+
+    # Set this outside the constructor because the constructor expects
+    # a json encodable, but this property will auto-update the json
+    # encodable field.
+    resolution.git_info = git_info
+    return resolution
```

Comment:
I think we have fixtures for this in `db/tests/fixtures.py`.

Reply:
I hadn't realized it could be used directly. Done.